File: LISTA 1/Multimetro.java
import java.util.Scanner;

public class Multimetro {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        final int tam = 10;
        double[] medicoes = new double[10];
        double desvioPadrao = 0, somaDvp = 0, media = 0;
        int i;

        // Lendo as medições e calculando a média
        for (i = 0; i < tam; i++) {
            System.out.println("Digite o valor da medicao " + (i + 1) + ": ");
            medicoes[i] = scanner.nextFloat();
            media += medicoes[i];
        }

        // calculando a média dos valores informados
        media /= tam;

        // Calculando a soma dos desvios quadrados e o desvio padrão
        for (i = 0; i < tam; i++) {
            somaDvp += Math.pow(medicoes[i] - media, 2);
        }

        // calculando o desvio padrão
        desvioPadrao = Math.sqrt(somaDvp / tam);

        // Imprimindo os resultados e verificando se o multimetro está funcionando corretamente
        System.out.println("A media eh: " + media);
        System.out.println("O desvio padrao eh: " + desvioPadrao);
        if (desvioPadrao > (media * 0.1)) {
            System.out.println("O multimetro esta com problemas e nao pode ser utilizado!");
        } else {
            System.out.println("O multimetro esta em estado normal de funcionamento!");
        }
    }
}

File: LISTA 2/multimetro.py
#Nome: Pedro Elias Santiago Mattar

import math

def main():
    tam = 10; desvioPadrao = 0; somaDvp = 0; media = 0.0; valor = 0.0
    medicoes = []
    
    #Lendo as medições e calculando a média
    for i in range(tam):
        valor = float(input("Digite o valor da medicao {}: ".format(i+1)))
        media+=valor
        medicoes.append(valor)
        
    #calculando a média dos valores informados
    media/=tam

    #Calculando a soma dos desvios quadrados e o desvio padrão
    for i in range(tam):
        somaDvp += pow(medicoes[i] - media, 2)

    #calculando o desvio padrão
    desvioPadrao = math.sqrt(somaDvp / tam)

    print("A media eh: ", media)
    print("O desvio padrao eh: ", desvioPadrao)

    #Imprimindo os resultados e verificando se o multimetro está funcionando corretamente
    if (desvioPadrao > (media * 0.1)) :
        print("O multimetro esta com problemas e nao pode ser utilizado!")
    else :
        print("O multimetro esta em estado normal de funcionamento!")    

main()